taskrun/ProcessTask.py

```diff
@@ -187,24 +195,24 @@ def execute(self):
       stderr_fd.close()
 
     # check the return code
-    ret = self._proc.returncode
-    if ret == 0:
+    self.retcode = self._proc.returncode
+    if self.retcode == 0:
       return None
     else:
-      return ret
+      return self.retcode
```

Comment:
why not returncode like Popen, so it's easier to guess?

Reply:
Done